.docker/README.md

```diff
@@ -0,0 +1,78 @@
+   ```sh
+   gcloud auth configure-docker
+   ```
+   
+**Note:** When using `sudo` with the docker commands, the gcloud configuration command in step 5, must also be entered with `sudo`.
+
+
+### Pull HarbourBridge's docker image
+
+Pull the [pre-built docker image](https://pantheon.corp.google.com/gcr/images/cloud-spanner-intern/global/harbour-bridge?project=cloud-spanner-intern) of HarbourBridge.
+```sh
+docker pull gcr.io/cloud-spanner-intern/harbour-bridge
+```
+### Run the image
+
+Run the image to perform migrations. The commands for it are described in this section.
+
+#### Migration using dump files
+
+The run command for migration using dump files is as follows.
+```sh
+docker run --rm \
+-v "$HOME/.config/gcloud:/gcp/config:ro" \
+-v /gcp/config/logs \
+--env CLOUDSDK_CONFIG=/gcp/config \
+--env GOOGLE_APPLICATION_CREDENTIALS=/gcp/config/application_default_credentials.json \
+-v "<path-to-directory-with-dump-files>:/harbourbridge/source_dump" \
+-v "<path-to-directory-to-store-output-files>:/harbourbridge/harbour_bridge_output" \
+gcr.io/cloud-spanner-intern/harbour-bridge:latest ../bin/harbourbridge \
+<migration-command-for-dump-files>
+```
+
+- The `<migration-command-for-dump-files>` for the different source databases can be found [here](https://github.com/cloudspannerecosystem/harbourbridge#running-harbourbridge).
+
+- The file paths in the `<migration-command-for-dump-files>` must be specified via `-source-profile` when running via docker and should be of the form - 
+```sh
+-source-profile="file=/harbourbridge/source_dump/<dump-file-name>"
+```
+
+#### Migration using direct-connection with hosted databases
+
+The run command for migration using direct connection with hosted databases is as follows.
+```sh
+docker run --rm \
+-v "$HOME/.config/gcloud:/gcp/config:ro" \
+-v /gcp/config/logs \
+--env CLOUDSDK_CONFIG=/gcp/config \
+--env GOOGLE_APPLICATION_CREDENTIALS=/gcp/config/application_default_credentials.json \
+-v "<path-to-directory-to-store-output-files>:/harbourbridge/harbour_bridge_output" \
+gcr.io/cloud-spanner-intern/harbour-bridge:latest ../bin/harbourbridge \
+<migration-command-for-direct-connect>
+```
+
+- The `<migration-command-for-direct-connect>` for the different source databases can be found [here](https://github.com/cloudspannerecosystem/harbourbridge#running-harbourbridge).
+
+<ins>**Key**<ins>
+   
+1. `<path-to-directory-with-dump-files>` :  The absolute path to the directory that has the dump files.
+2. `<path-to-directory-to-store-output-files>` : The absolute path to the directory that is to store the output files generated by HarbourBridge.
+
+<ins>**Note:**</ins>
+
+1. The `<migration-command-for-dump-files>` should **not** include `go run github.com/cloudspannerecosystem/harbourbridge`.
+2. The `<migration-command-for-direct-connect>` should **not** include `go run github.com/cloudspannerecosystem/harbourbridge`.
```

Comment:
This is confusing. Can you instead give examples of `schema`, `schema-and-data` and `data` command?

You can create a separate section `Examples` and provide the examples to the reader.

Reply:
Okay.

Reply:
> You can create a separate section Examples and provide the examples to the reader.

Added example run commands for the `schema-and-data` mode. Adding examples for `schema` mode and `data` mode as well would clutter the README. 

We've mentioned that they can be used with the same command paradigm.